docs/updates/2019-01.md

```diff
@@ -0,0 +1,44 @@
+# January 2019
+
+New year, new features, new solutions. In this update we will share some news about our latest tool releases.
+
+## Data model experience
+
+Our data modelling tool, catwalk, has finally reached a maturity level where we feel comfortable to move it from an
+experimental state. Since it has continuously been published on [catwalk.core.qlik.com](https://catwalk.core.qlik.com)
+you will not feel lost if you visited before, but here are some of our latest additions:
+
```

Comment:
I would remove "finally", and rephrase it a bit.

Maybe sth like "Our data modelling tool catwalk has been improved and will move from being experimental to stable. Since it continuously has been deployed on [catwalk.core.qlik.com](https://catwalk.core.qlik.com) you will, if you have visited catwalk before, already feel familiar with the look and feel. Here are some of our latest additions:"

Reply:
👍